modules/services/agentless-scanning/main.tf

```diff
@@ -50,6 +50,31 @@ data "aws_iam_policy_document" "scanning" {
     ]
   }
 
+  statement {
```

Comment:
Few formatting comments in the `Data` block in this file main.tf.
Terraform Data module doesn't allow ` , (comma)` after `sid`, `action`, `resource` argument definitions.

Reply:
this is fixed